geometry_msgs/Pose2D robot
geometry_msgs/Pose2D ally1
geometry_msgs/Pose2D ally2
geometry_msgs/Pose2D opponent1
geometry_msgs/Pose2D opponent2

geometry_msgs/Point ball